web/dashboard_info.js

```diff
@@ -1202,6 +1209,63 @@ netdataDashboard.context = {
             }
         ]
     },
+    
+    // ------------------------------------------------------------------------
+    // beanstalkd
+    // system charts
+    'beanstalk.cpu_usage': {
+        info: 'Amount of CPU Time for user and system used by beanstalkd'
+    },
+
+    // This is also a per-tube stat
+    'beanstalk.jobs_rate': {
+        info: 'Number of jobs processed per second'
+    },
+
+    'beanstalk.connections_rate': {
+        info: 'Number of connections opened per second'
+    },
+
+    'beanstalk.commands_rate': {
+        info: 'Number of commands per second, group by command name'
+    },
+
+    'beanstalk.current_tubes': {
+        info: 'Total number of current tubes on the server including the default tube (which always exists)'
+    },
+
+    'beanstalk.current_jobs': {
+        info: 'Current number of jobs in all tubes grouped by status: urgent, ready, reserved, delayed and buried'
+    },
+
+    'beanstalk.current_connections': {
+        info: 'Current number of connections group by connection type: written, producers, workers, waiting'
+    },
+
+    'beanstalk.binlog': {
+        info: 'Written: Cumulative number of records writen to the binlog<br> Migrated: Cumulative number of records written as part of the compaction'
+    },
+
+    'beanstalk.uptime': {
+        info: 'Total time beanstalkd server has been up for'
+    },
+
+    // tube charts
+    'beanstalk.jobs': {
+        info: 'Cumulative number of jobs currently in the tube grouped by status: urgent, ready, reserved, delayed and buried'
+    },
+
+    'beanstalk.connections': {
+        info: 'Total number of connections to this tube grouped by connection type; using, waiting and watching'
```

Comment:
I think we should turn `Total` to `The` (total = cumulative? This is is not cumulative)